nixos/modules/services/games/factorio.nix

```
@@ -87,6 +87,18 @@ in
           a new map with default settings will be generated before starting the service.
         '';
       };
+      loadLatestSave = mkOption {
+        type = types.bool;
+        default = true;
```

This is technically a breaking change. Also introducing "impure" behavior feels like this is becoming less declarative, and stateful.

cc @SuperSandro2000

---

```suggestion
        default = false;
```
I think the default and what is most mentioned in guides is to load a specific save.

---

Most guides assume vanilla, I imagine, which never crashes. But I'm happy to leave it off by default if you think it'll confuse people.